specification/paths/Shipments-shipment_id.json

```diff
@@ -49,6 +49,59 @@
       }
     }
   },
+  "patch": {
+    "tags": [
+      "Shipments"
+    ],
+    "summary": "Update a shipment by id.",
+    "description": "This endpoint updates the shipment with the supplied id.",
+    "parameters": [
+      {
+        "$ref": "#/components/parameters/path-shipment_id"
+      }
+    ],
+    "requestBody": {
+      "description": "The shipment object with attributes to be updated.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "data": {
+                "$ref": "#/components/schemas/Shipment"
```

Comment:
Won't this require the user to always supply the entire shipment when patching?

Reply:
Yup, this is the case for all `PATCH` endpoints, probably introduced with the OpenApi migration. I created task MP-621 to fix this implementation.